modules/swagger-codegen/src/main/resources/lua/api.mustache

```diff
@@ -27,28 +28,50 @@ local function new_{{classname}}(host, basePath, schemes)
     basePath = basePath or "{{{basePath}}}";
     schemes = schemes_map;
     default_scheme = default_scheme;
+    http_username = '';
+    http_password = '';
+    api_key = {}
+    access_token = '';
   }, {{{packageName}}}_mt)
 end
 
+// for base64 encoding
+local function base64_encode(s)
```

Comment:
Use `basexx.to_base64`. See https://github.com/daurnimator/lua-http/blob/master/http/request.lua#L85 for an example